src/commonMain/kotlin/character/races/abilities/Berserking.kt

```diff
@@ -39,7 +39,8 @@ class Berserking : Ability() {
 
         override fun modifyStats(sp: SimParticipant): Stats {
             return Stats(
-                physicalHasteMultiplier = 1.1
+                physicalHasteMultiplier = 1.1,
+                spellHasteRating = 10 * Rating.hastePerPct
```

Comment:
This should probably modify `spellHasteMultiplier`, since Berserking would have a multiplicative effect with other independent sources of haste (like cumulative spell haste rating).

Reply:
I thought the mechanics for the multiplier would only use existing haste rating, but upon further review it looks to default to base 1.0 as the base for spell haste so will change this.